### PR TITLE
[WIP] feat: Add nv22 skeleton

### PR DESCRIPTION
**Pull request by TippyFlitsUK**
Addition of Network Version 22 skeleton

### CODE REVIEW

**Comment by aarshkshah1992** [comment]
@TippyFlitsUK What script did you use to generate the changes in this PR ?

**Comment by TippyFlitsUK** [comment]
I was following the guide [here](https://github.com/filecoin-project/lotus/blob/9cf748215c001d641a3584e070bfb6d4b1ef8b1b/documentation/misc/Building_a_network_skeleton.md) @aarshkshah1992 👍 
I believe it was the `make gen` step.

File: network/version.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ const (
 	Version19                 // Lightning
 	Version20                 // Thunder
 	Version21                 // SyntheticPoRep
+	Version22                 // DDO
```
Comment (arajasek) [inline network/version.go:32]: Nit: Call this TBD, and call the line above Watermelon?

File: builtin/v13/migration/miner.go [hunk 2]
```diff
@@ -0,0 +1,364 @@
+package migration
```
Comment (arajasek) [inline builtin/v13/migration/miner.go:1]: This file is specific to the nv21 migration. I would propose dropping the entire `migration` directory for now -- we'll have to add the "lightweight" migration, but we can land this PR without that.

**Review by arajasek — COMMENTED** [review]
Just a couple nits.

**Review by arajasek — APPROVED** [review]

**Pull request merged**